src/pages/office/cat-cpc/office.cat-cpc.page.html

```diff
@@ -33,36 +43,59 @@ <h4>Test summary</h4>
+              [formGroup]="form"
+              [debriefWitnessed]="pageState.debriefWitnessed$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (debriefWitnessedChange)="debriefWitnessedChanged($event)">
+            </debrief-witnessed>
+
+            <candidate-description
+              *ngIf="!(isDelegated && !isPass())"
+              [display]="(pageState.displayCandidateDescription$ | async)"
+              [formGroup]="form"
+              [candidateDescription]="pageState.candidateDescription$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (candidateDescriptionChange)="candidateDescriptionChanged($event)">
             </candidate-description>
 
-            <identification [display]="pageState.displayIdentification$ | async" [formGroup]="form"
-                            [identification]="pageState.identification$ | async"
-                            [outcome]="pageState.testOutcome$ | async"
-                            (identificationChange)="identificationChanged($event)">
+            <identification
+              *ngIf="!(isDelegated && !isPass())"
+              [display]="(pageState.displayIdentification$ | async)"
+              [formGroup]="form"
+              [identification]="pageState.identification$ | async"
+              [outcome]="pageState.testOutcome$ | async"
+              (identificationChange)="identificationChanged($event)">
             </identification>
 
-            <combination [combination]="pageState.combination$ | async"
-                         [combinationAdditionalText]="getCombinationAdditionalText(combinationCode)">
-            </combination>
-
             <additional-information [display]="pageState.displayAdditionalInformation$ | async"
                                     [additionalInformation]="pageState.additionalInformation$ | async"
                                     [formGroup]="form"
                                     [outcome]="pageState.testOutcome$ | async"
                                     (additionalInformationChange)="additionalInformationChanged($event)">
             </additional-information>
 
-            <assessment-report *ngIf="displayIfFail(outcome)"
+            <combination
+              [combination]="pageState.combination$ | async"
+              [combinationAdditionalText]="getCombinationAdditionalText(combinationCode)">
+            </combination>
+
+            <assessment-report *ngIf="isFail() && !isDelegated"
                                [assessmentReport]="pageState.assessmentReport$ | async"
                                [formGroup]="form"
                                (assessmentReportChange)="assessmentReportChanged($event)">
             </assessment-report>
 
-            <cpc-debrief-card *ngIf="displayIfFail(outcome) && pageState.combination$ | async"
+            <cpc-debrief-card *ngIf="isFail() && pageState.combination$ | async"
```

Comment:
looks like debrief witnessed is needed on all 3 outcomes (from miro), wont isFail cause it to be hidden

Reply:
this isn't debrief witnessed and i didnt change the logic here just removed the necessity for a parameter and changed the name of it to something more appropriate

Reply:
besides this is from the rebase against fail journey, not going to be part of this pr when it gets merged in